app/scripts/lib/transaction/metrics.ts

```diff
@@ -1000,7 +1008,9 @@ async function buildEventFragmentProperties({
     // ui_customizations must come after ...blockaidProperties
     ui_customizations: uiCustomizations.length > 0 ? uiCustomizations : null,
     transaction_advanced_view: isAdvancedDetailsOpen,
-    transaction_contract_method: transactionContractMethod,
+    transaction_contract_method: transactionContractMethod
```

Comment:
On this and `transaction_contract_address`, should the properties names be plural because it's an array (even if it's only at most a one element array)?

Reply:
I assume we want to keep it the same for consistency and existing dashboard etc.